tests/go/GofmtBearTest.py

```diff
@@ -1,7 +1,64 @@
+from queue import Queue
+
 from bears.go.GofmtBear import GofmtBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+test_file1 = """package main
```

Comment:
maybe rename these to `in_file1`, `out_file1`, `simplify_in_file1`, `simplify_out_file1` to make it a bit clearer what is happening.